src/FormatNumber/Parser.elm

```diff
@@ -106,6 +106,41 @@ splitThousands integers =
         |> reversedSplitThousands
         |> List.reverse
 
+{-| Split a `String` in `List String` grouping by digits as per Indian 
+numbering system. Last 3 digits are grouped together but after that 
+numbers are grouped in two. 
+[Indian numbering system](https://en.wikipedia.org/wiki/Indian_numbering_system#Use_of_separators):
+
+    splitByIndian "12345678" --> [ "1", "23", "45", "678" ]
+
+    splitByIndian "12" --> [ "12" ]
+
+-}
+splitByIndian : String -> List String
+splitByIndian integers =
```

Comment:
For consistency, should we rename `splitThousands` as `splitByWestern`?

Reply:
Yes, it can be template if we have to add support for more numbering systems.